packages/vike-vue/src/renderer/onRenderHtml.ts

```diff
@@ -41,6 +43,14 @@ const onRenderHtml: OnRenderHtmlAsync = async (pageContext): ReturnType<OnRender
     headHtml = dangerouslySkipEscape(await renderToStringWithErrorHandling(app))
   }
 
+  const bodyHtmlStart = dangerouslySkipEscape(pageContext.config.bodyHtmlStart?.(pageContext) ?? '')
+
+  const bodyHtmlEnd = dangerouslySkipEscape(
```

Comment:
I would go with a default empty `bodyHtmlEnd` - I suppose some don't use teleport, and for the ones that use it the probability of teleporting to a `#teleported` target is small :)
So people would need to define a custom `bodyHtmlEnd` in config most of the time.

Reply:
My thought was to provide a default target and make it clear in the docs. That way users wouldn't need to provide their own `bodyHtmlEnd` just to use teleports.